### DISCUSSION TITLE
Extension

### DISCUSSION THREAD

**Discussion by creeperbommyyx** [3kh0/ext-remover#198] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is it possible to make a bookmarklet to install a blocked extension, as there being developer mode blocked?

**Comment by Fallenmoon8080** [comment]
This is unfortunately not possible at the moment.

**Comment by Sxri** [comment, marked as answer]
Not that I know of, there is a team of dev's working on resolving the patch in the disable extensions exploit, however, adding custom extensions is not possible at the moment.

**Comment by The-Man-Is-Me** [comment]
Not at the moment. LTBEEF exploited the fact the webstore did not properly check if the extension on/off signal was from the webstore. Even at version 105 it still properly checked where extensions came from. Look at this https://replit.com/@CompactCow/i-love-replit-it-is-my-favorite-website-ever-and-its-so-goo?v=1 for a possible solution. (It did not work for me, don't know. ) ps. found this on a discussion, can't find it now, whoever posted it, let me know.

**Comment by creeperbommyyx** [comment]
Is it possible for a bookmarklet to change/edit chrome policies?

**Reply by Sxri** [reply]
Not that I know of, if so, it would probably be patched by now.

**Comment by creeperbommyyx** [comment]
I have a chromebook on v105 btw

**Reply by ghost** [reply]
ok, then you can use ltbeef

**Comment by creeperbommyyx** [comment]
Or is it possible to change the managed without doing funky stuff like taking out the chromebook cover?

**Reply by ghost** [reply]
yeah, but you most DEFINITELY won't because it is illegal

**Comment by creeperbommyyx** [comment]
also linux terminal is blocked

**Comment by creeperbommyyx** [comment]
any way to use like chromebook developer mode to use like shell or something?
